cmd/wallet-web/src/pages/Vaults.vue

```diff
@@ -63,6 +70,27 @@ export default {
       const { contents } = await getCredential;
       this.credentialsFound = Object.keys(contents).length;
     },
+    fetchAllVaults: async function (token, collectionManager) {
+      const { contents } = await collectionManager.getAll(token);
+      console.log(`found ${Object.keys(contents).length} vaults`);
+
+      const collections = Object.keys(contents);
+
+      collections.map(async (id) => {
```

Comment:
You have to use "array.Map()" when you are building new array by using underlying array as reference.

In this case, you should use array.ForEach() or `this.vaults = collections.map(.....)` where collection map function returns vault (`return vault`)

Reply:
& `getAll` already returned you all the vaults in key-value pairs, why you have to hit the store again?